galley/pkg/config/analysis/analyzers/virtualservice/conflictingmeshgatewayhosts.go

```diff
@@ -37,7 +37,8 @@ var _ analysis.Analyzer = &ConflictingMeshGatewayHostsAnalyzer{}
 // Metadata implements Analyzer
 func (c *ConflictingMeshGatewayHostsAnalyzer) Metadata() analysis.Metadata {
 	return analysis.Metadata{
-		Name: "virtualservice.ConflictingMeshGatewayHostsAnalyzer",
+		Name:        "virtualservice.ConflictingMeshGatewayHostsAnalyzer",
+		Description: "Checks if multiple VirtualServices associated with mesh gateway have conflicting hosts.",
```

Comment:
Nit: let's make a quick pass to try to be consistent in the language used for descriptions. e.g. "virtual services" vs "VirtualServices".  Since this is now public-facing, we'll want to raise the standard a bit higher than we would for just code comments. @geeknoid is there a style guide or something we could refer to?

Reply:
The style guide is https://istio.io/about/contribute/style-guide/ and I recommend we use it for user-facing messages.

Reply:
I admit, I'm a bit confused after reading that about whether or not we should say "VirtualServices" or "virtual services".

> If you are not referencing values or code directly, use normal sentence capitalization, for example, “The Istio role binding configuration takes place in a YAML file.”

Reply:
Ended up going with the second ("virtual services"). In places where that's not feasible (e.g. the schema analyzers), I put quotes around the kind with the hopes that it's clear we're referring to a config resource kind.